documentation/API-design-guidelines.md

```diff
@@ -1326,25 +1328,52 @@ If this capability is present in CAMARA API, `webhook` object attribute **must**
 
 | attribute name | type | attribute description | cardinality |
 | ----- |	-----  |	 -----  | -----  | 
-| notificationUrl | string | https callback address where the notification must be POST-ed | mandatory |
-| notificationAuthToken | string | OAuth2 token to be used by the callback API endpoint. It MUST be indicated within HTTP Authorization header e.g. Authorization: Bearer $notificationAuthToken | optional |
+| sink | string | https callback address where the notification must be POST-ed | mandatory |
+| sinkCredential | object | Sink credential provides authentication or authorization information necessary to enable delivery of events to a target. In order to be updated in future this object is polymorphic | optional |
 
-##### Instance-based (implicit) subscription example
+Several types of `sinkCredential` could be available in future but for now only access token credential are managed.
 
-```javascript
+``sinkCredential`` attributes table (must be access token for now):
+
+| attribute name | type | attribute description | cardinality |
+| ----- |	-----  |	 -----  | -----  | 
+| credentialtype | string | Type of the credential - MUST be set to `ACCESSTOKEN` for now | mandatory |
+| accessToken | string | Access Token granting access to the POST operation to create notification | mandatory |
+| accessTokenExpireUtc | string date-time | An absolute UTC instant at which the access token shall be considered expired. | mandatory |
```

Comment:
One point that we need to discuss. When Access Token expires, How to manage the subscription

Reply:
I guess we need to terminate the subscription and provide a subscription-ends event with terminationReason set to "ACCESS_TOKEN_EXPIRED" ?  WDYT?

Reply:
Let me think and back here in this week.

Reply:
It is OK